beluga_system_tests/test/test_system.cpp

```diff
@@ -120,7 +120,7 @@ TEST_P(BelugaSystemTest, testEstimatedPath) {
       }));
   for (const auto [iteration, data_point] : ranges::views::enumerate(test_data.data_points)) {
     auto scan_copy = data_point.scan;
-    const auto updated_pose = pf.update_filter(data_point.odom, std::move(scan_copy));
+    const auto updated_pose = pf.update_filter(data_point.odom, std::move(scan_copy), false);
```

Comment:
@glpuga nit^2: consider using a named constant for readability sake:
```suggestion
    constexpr bool kForceUpdate = true;
    const auto updated_pose = pf.update_filter(data_point.odom, std::move(scan_copy), !kForceUpdate);
```